build_steps/deploy.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+git add Dockerrun.aws.json
+
+if [ "$?" != "0" ]; then
+  exit 1
+fi
+
+docker run --rm \
+  -v $(pwd):/home \
+  -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
+  -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION \
+  manuelgfx/docker-aws-eb-cli-git \
+  use $PROJECT_NAME-$APP_ENV
+
+if [ "$?" != "0" ]; then
+  exit 1
+fi
+
+docker run --rm \
+  -v $(pwd):/home \
+  -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
+  -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION \
+  manuelgfx/docker-aws-eb-cli-git \
+  deploy --staged
+
+if [ "$?" != "0" ]; then
+  exit 1
+fi
+
+docker run --rm \
+  -v $(pwd):/home \
+  -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID \
+  -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY \
+  -e AWS_DEFAULT_REGION=$AWS_DEFAULT_REGION \
+  manuelgfx/docker-aws-eb-cli-git \
```

Comment:
See my comments relating to using an external source for docker/aws cli commands